.github/workflows/pipelinit.ruby.lint.yaml

```diff
@@ -0,0 +1,19 @@
+# Generated with pipelinit 0.2.2
+# https://pipelinit.com/
+name: Lint Ruby
+on:
+  pull_request:
+    paths:
+      - "**.rb"
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "1.9.3
+"
```

Comment:
This line-break before the quotation marks can break the file configuration.

Tip:
Check if the file presents any error with the `actionlint` command
- Install it: https://github.com/rhysd/actionlint#quick-start
- Then run against this file `actionlint .github/workflows/pipelinit.ruby.lint.yaml`